lib/resource.nunjucks

```diff
@@ -132,26 +132,40 @@
                       <h3>Body</h3>
                       {% for key, b in method.body %}
                         <p><strong>Media Type: {{ key }}</strong></p>
-                        {% if b.formParameters %}
+                        {% if b.formParameters or b.type and not b.schema %}
```

Comment:
L163 contains a check for the type property already, and here as well. I'd recommend putting your types rendering under L163. Checking here for `formParameters` is only for compatibility to RAML 0.8.